### PR TITLE
Updates for spelling, some tiny but obvious grammar, punctuation omis…

### PR DESCRIPTION
**Pull request by amy-durocher**
[READY FOR REVIEW] Updates for spelling, some tiny but obvious grammar, punctuation omissions. Also made a few branding string updates per branding guidelines.

@lrosenth @mprabhak please review and approve

### CODE REVIEW

File: schemas/context/experienceevent-enduserids.schema.json [hunk 1]
```diff
@@ -7,19 +7,19 @@
   ],
   "$id": "https://ns.adobe.com/xdm/context/experienceevent-enduserids",
   "$schema": "http://json-schema.org/draft-06/schema#",
-  "title": "ExperienceEvent endUserIDs",
+  "title": "ExperienceEvent EndUserIDs",
```
Comment (K-Duncan) [inline schemas/context/experienceevent-enduserids.schema.json]: Check - I think endUserIDs might be correct.

File: schemas/data/datasource.schema.json [hunk 2]
```diff
@@ -9,15 +9,15 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "Data Source",
   "type": "object",
-  "description": "The Datasource acts as a namespace or unique identifier associated with a collection of data. Each EndUserID is associated with a given Datasource. A user can create and obtain information about a Datasource during the solution onboarding.\n",
+  "description": "The Data source acts as a namespace or unique identifier associated with a collection of data. Each EndUserID is associated with a given data source. A user can create and obtain information about a data source during the solution onboarding.\n",
   "definitions": {
```
Comment (K-Duncan) [inline schemas/data/datasource.schema.json]: Data source or data source?

File: schemas/context/experienceevent-enduserids.schema.json (hunk 1 above)
Comment (amy-durocher) [reply schemas/context/experienceevent-enduserids.schema.json]: Fixed!

File: schemas/data/datasource.schema.json (hunk 2 above)
Comment (amy-durocher) [reply schemas/data/datasource.schema.json]: Definitely data source in this context.  You have an eagle eye!

**Comment by prabhum2** [comment]
@cfraser - Did you review this . Can we go ahead and merge ?

**Comment by cfraser** [comment]
LGTM

**Pull request merged**